autoscheduler/frontend/src/components/SchedulingPage/SchedulingPage.tsx

```diff
@@ -21,6 +22,14 @@ const SchedulingPage: React.FC<RouteComponentProps> = (): JSX.Element => {
     });
   }, [dispatch]);
 
+  // get current term
+  const termCurr = useSelector<RootState, string>((state) => state.term);
```

Comment:
Could you move this `termCurr` definition to right below the `const dispatch = `? We typically like to have the "state" declaring before like taking actions/calling things (e.g. the `React.useEffect` or your `return null` when the term is fasly)

Reply:
Fixed with the revert to redirect flashing.